packages/terra-application-name/examples/index-examples/ApplicationHeaderNameStandard.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import ApplicationHeaderLayout from 'terra-application-header-layout';
+import Image from 'terra-image';
+import ResponsiveElement from 'terra-responsive-element';
+import NavigationExample from 'terra-application-header-layout/examples/index-examples/NavigationExample';
+import ToggleExample from 'terra-application-header-layout/examples/index-examples/ToggleExample';
+import UtilitiesExample from 'terra-application-header-layout/examples/index-examples/UtilitiesExample';
+import ExtensionsExample from 'terra-application-header-layout/examples/index-examples/ExtensionsExample';
+import ApplicationHeaderName from '../../lib/ApplicationHeaderName';
+
+const ApplicationHeaderNameStandard = () => {
+  const color = { color: 'grey' };
```

Comment:
I think this is just used on one place. If that's so, I would leave out this variable.

Reply:
Do this throughout the rest of your examples